01. installation/04. webservers.md

```diff
@@ -41,7 +41,7 @@ url.rewrite-if-not-file = ( "^/?$" => "$0",
 }
 ```
 
-In this sample configuration, Fork CMS needs to rewrite its urls. This rewrite rule is based on the Apache .htaccess-file and should work with the latest lighttpd 1.4.x without causing a redirect loop.
+In this sample configuration, Fork CMS needs to rewrite its urls. This rewrite rule is based on the Apache .htaccess-file and should work with the latest Lighttpd 1.4.x without causing a redirect loop.
```

Comment:
Q: It's 2016. Does anyone still use Lighttpd or Cherokee? It's all apache or nginx in the world of webservers. 
Maybe we should keep it simple and remove that part from the docs?